docs/tutorial.rst

```diff
@@ -212,4 +209,4 @@ With very little data, rasa NLU can in certain cases already generalise concepts
     }
 
 even though there's nothing quite like this sentence in the examples used to train the model. 
-To build a more robust app you will obviously want to use a lot more data, so go and collect it!
+To build a more robust app you will obviously want to use a lot more training data, so go and collect them!
```

Comment:
let's keep the "it" (I know there is a heated debate about it - but let's stay consistent across the documentation)